internal/securitate/HttpPutOrder.go

```diff
@@ -46,8 +46,8 @@ func (dataBase *DBstruct) PutOrder(rwr http.ResponseWriter, req *http.Request) {
 		models.Sugar.Debugf("422 — неверный формат номера заказа; %d\n", orderNum)
 		return
 	}
-	var orderID int64
-	err = dataBase.GetIDByOrder(req.Context(), orderNum, &orderID)
+//	var orderID int64
+	orderID, err := dataBase.GetIDByOrder(req.Context(), orderNum)
```

Comment:
```suggestion
	storedUserID, err := dataBase.GetUserIDByOrder(req.Context(), orderNum)
```
иначе дальше сравнение `orderID == userID` выглядит, как ошибка. Да и именование функции стоит изменить, требования к читабельности и поддерживаемости кода никто не отменял.